### PR TITLE
v1.0.0

### PR DESCRIPTION
**Pull request by ronuma**
Chat fixes, queue 100%, ccp login popup, ui fixes

### CODE REVIEW

File: src/components/chat/chatBot/index.tsx [hunk 1]
```diff
@@ -83,7 +83,7 @@ const ChatBot: React.FC = () => {
     if (!isOpen) return
     if (!profileID) return
     const WEBSOCKET_ENDPOINT = process.env.NEXT_PUBLIC_WEBSOCKET_ENDPOINT
-    ws.current = new WebSocket(`wss://${WEBSOCKET_ENDPOINT}?agentID=${profileID}&secondaryID=copilot`)
+    ws.current = new WebSocket(`ws://${WEBSOCKET_ENDPOINT}?agentID=${profileID}&secondaryID=copilot`)
```
Comment (ronuma) [inline src/components/chat/chatBot/index.tsx:86]: why tho?

File: src/components/chat/textBox/index.tsx [hunk 2]
```diff
@@ -24,17 +24,17 @@ const TextBox: React.FC<TextBoxProps> = ({ isAgent = false, agent = {}, agentID
...
+  useMemo(() => {
     const initializeWebSocket = async () => {
       const WEBSOCKET_ENDPOINT = process.env.NEXT_PUBLIC_WEBSOCKET_ENDPOINT
       if (!WEBSOCKET_ENDPOINT) {
         console.log('WEBSOCKET_ENDPOINT not found')
         return
       }
       if (isAgent) {
-        ws.current = new WebSocket(`wss://${WEBSOCKET_ENDPOINT}?agentID=${agentID}&secondaryID=supervisor`)
+        ws.current = new WebSocket(`ws://${WEBSOCKET_ENDPOINT}?agentID=${agentID}&secondaryID=supervisor`)
```
Comment (ronuma) [inline src/components/chat/textBox/index.tsx:35]: this will break in deployment!!

File: src/components/chat/textBox/index.tsx [hunk 3]
```diff
@@ -24,17 +24,17 @@ const TextBox: React.FC<TextBoxProps> = ({ isAgent = false, agent = {}, agentID
...
       } else {
-        ws.current = new WebSocket(`wss://${WEBSOCKET_ENDPOINT}?agentID=${selectedAgent.id}&secondaryID=supervisor`)
+        ws.current = new WebSocket(`ws://${WEBSOCKET_ENDPOINT}?agentID=${selectedAgent.id}&secondaryID=supervisor`)
```
Comment (ronuma) [inline src/components/chat/textBox/index.tsx:37]: and this

File: src/app/(main)/@supervisor/chat/page.tsx [hunk 4]
```diff
@@ -23,16 +22,19 @@ function Chat() {
...
           filter: {
-            notification_type: { eq: NotificationType.HUMAN },
+            notification_type: { eq: NotificationType.AGENT },
             read: { eq: false },
           },
         },
       })
-      const notifications = notificationsRes.data.listNotifications.items
+      notifications = notificationsRes.data.listNotifications.items
+      console.log('notifications', notifications)
```
Comment (tarazonaa) [inline src/app/(main)/@supervisor/chat/page.tsx:37]: this one needs to go mb

**Review by tarazonaa — APPROVED** [review]
Fire stuff imo we be takin Ws left and right 😈

**Pull request merged**